README.md

```diff
@@ -212,6 +212,10 @@ pyblish_lite.settings.InitialTab = "overview"
 # Customize whether to use labels for plugins and instances.
 # Default: True
 pyblish_lite.settings.UseLabel = False
+
+# Custommize the width and height of the window
+pyblish_lite.settings.WindowWidth = 500
```

Comment:
Let's keep it similar to pyblish_qml.

- https://github.com/pyblish/pyblish-qml/blob/master/pyblish_qml/settings.py#L7